### PR TITLE
feat: added new banner to cfd cards

### PR DESCRIPTION
**Pull request by shontzu-deriv**
## Changes:

the "New!" banner is done, for demonstration purpose, it is applied on all cfd cards until derivez and ctrader is displayed
in the PR, condition is applied to only show the banner for derivez and ctrader

### Screenshots:

image

### CODE REVIEW

**Review by hamza-deriv — APPROVED** [review]

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import CFDInstrumentsLabelHighlighted from './cfd-instruments-label-highlighted'
 import CFDCompareAccountsDescription from './cfd-compare-accounts-description';
 import CFDCompareAccountsTitleIcon from './cfd-compare-accounts-title-icon';
 import CFDCompareAccountsPlatformLabel from './cfd-compare-accounts-platform-label';
+import { Text } from '@deriv/components';
```
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx]: Put the '@deriv/components' imports on top of the the other imports.
The order should be: 1. external libraries 2. deriv packages 3. internal components

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx [hunk 2]
```diff
@@ -12,8 +14,14 @@ type TCFDCompareAccountsCardProps = {
 const CFDCompareAccountsCard = ({ trading_platforms }: TCFDCompareAccountsCardProps) => {
     return (
         <div className='compare-cfd-account-main-container'>
-            <CFDCompareAccountsPlatformLabel trading_platforms={trading_platforms} />
             <div className='compare-cfd-account-card-container'>
+                <CFDCompareAccountsPlatformLabel trading_platforms={trading_platforms} />
+                {(trading_platforms.platform === CFD_PLATFORMS.DERIVEZ ||
+                    trading_platforms.platform === CFD_PLATFORMS.CTRADER) && (
+                    <Text className='compare-cfd-account-card-container__banner' weight='bold' size='xs'>
+                        New!
```
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx]: use localize

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss [hunk 3]
```diff
@@ -29,13 +29,29 @@
...
         &:hover {
             box-shadow: 0 2px 8px 0 var(--shadow-menu);
         }
+        &__banner {
+            position: absolute;
+            z-index: 50;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            padding: 5px 10px;
```
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss]: use rem

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss [hunk 4]
```diff
@@ -29,13 +29,29 @@
...
+            width: 200px;
```
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss]: same for the rest of these

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss [hunk 5]
```diff
@@ -29,13 +29,29 @@
...
         width: 27rem;
         border: 1px solid var(--general-hover);
-        padding: 2.4rem;
         border-radius: 2.4rem;
         &:hover {
             box-shadow: 0 2px 8px 0 var(--shadow-menu);
         }
+        &__banner {
+            position: absolute;
+            z-index: 50;
```
Comment (hirad-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss]: I suggest using 9999 to have it on top of everything which might be added later. either the highest number or a 1 digit number. number 50 is so randomly chosen

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx [hunk 6]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { Text } from '@deriv/components';
 import { CFD_PLATFORMS } from '@deriv/shared';
 import { Localize } from '@deriv/translations';
-import { TModifiedTradingPlatformAvailableAccount } from 'Components/props.types';
+import { TModifiedTradingPlatformAvailableAccount, TCompareAccountsCard } from 'Components/props.types';
```
Comment (hamza-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-card.tsx]: TModifiedTradingPlatformAvailableAccount is not usable anymore in the file, so kindly remove it

**Review by hirad-deriv — APPROVED** [review]

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss [hunk 7]
```diff
@@ -29,13 +29,29 @@
...
+        &__banner {
+            position: absolute;
+            z-index: 1000;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            padding: 1.5rem;
+            width: 13rem;
+            height: 2rem;
+            right: -4rem;
```
Comment (hamza-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss]: we usually dont use the right left in (-ve) kindly use other css property 
Its a vague value

File: packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss [hunk 8]
```diff
@@ -29,13 +29,29 @@
...
         width: 27rem;
         border: 1px solid var(--general-hover);
-        padding: 2.4rem;
         border-radius: 2.4rem;
         &:hover {
             box-shadow: 0 2px 8px 0 var(--shadow-menu);
         }
+        &__banner {
+            position: absolute;
+            z-index: 1000;
```
Comment (hamza-deriv) [inline packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.scss]: use 9999 or 1
```suggestion
            z-index: 1;
```

**Review by hamza-deriv — APPROVED** [review]

**Pull request merged**